source/ways-of-working/new-component/ssh-via-vpn.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+title: Set up SSH via the VPN
```

Comment:
This should be about anything via VPN, not just SSH, e.g 443 is the most common one we do